tools/tripal/analysis_load_blast.xml

```diff
@@ -117,8 +87,6 @@
 
         The `tripal_analysis_blast <http://github.com/tripal/tripal_analysis_blast>`_ module must be installed and configured on the Tripal instance.
 
-        @HELP_JOB@
-
         @HELP@
```

Comment:
ooh, we should add this to command engine, generic templating stuff, that would be usfeul.